src/where.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import {broadcast, getBroadcastShape} from './lib/broadcast.js';
+import {Tensor, sizeOfShape, Scalar} from './lib/tensor.js';
+
+/**
+ * Select the values from the input or the other tensor depending on
```

Comment:
Please update this explanation using changed **trueValues** **and falseValues**.

Reply:
OK